src/components/Contexts/Variables/Modals/OdbImport/OdbImport.tsx

```diff
@@ -23,17 +23,17 @@ export function OdbImport() {
   const rotator = useRotator().data?.rotator;
 
   function updateObs() {
-    updateConfiguration({
+    void updateConfiguration({
```

Comment:
Just to learn :sweat_smile:
Why is this `void` needed?

Reply:
Good question! I updated the linting rules to not allow "floating promises" which are promise-returning functions that are never awaited. It's a great rule that can catch bugs, but in cases like these where it's an event handler we have to "discard" the promise (void)

Reply:
Awesome, thank you for the explanation.